government/borders.json

```diff
@@ -50,13 +48,21 @@ permalink: /government/borders.json
+                [[-7330,-2512],[-7310,-2655],[-7249,-2680],[-7179,-2670],[-7152,-2612],[-7115,-2584],[-7053,-2552],[-6979,-2538],[-6960,-2425],[-7012,-2348],[-7066,-2298],[-7159,-2297],[-7273,-2320],[-7286,-2369],[-7324,-2397]],
+                [[-1701,-6351],[-1696,-6301],[-1732,-6271],[-1724,-6239],[-1697,-6190],[-1665,-6190],[-1629,-6148],[-1595,-6125],[-1594,-6086],[-1613,-6074],[-1635,-6065],[-1661,-6070],[-1680,-6049],[-1662,-6027],[-1652,-6028],[-1632,-6040],[-1613,-6022],[-1544,-5981],[-1502,-5930],[-1527,-5849],[-1482,-5730],[-1649,-5547],[-1775,-5484],[-1890,-5597],[-1999,-5681],[-2022,-5827],[-2031,-5985],[-2032,-6113],[-2025,-6245],[-2006,-6328],[-1917,-6359],[-1881,-6393],[-1864,-6446],[-1836,-6473],[-1845,-6513],[-1833,-6532],[-1806,-6525],[-1780,-6510],[-1768,-6494],[-1776,-6465],[-1746,-6445],[-1737,-6423],[-1708,-6394]],
+                [[-5298,765],[-5400,700],[-5461,752],[-5487,789],[-5505,828],[-5533,876],[-5507,946],[-5398,1017],[-5299,952],[-5230,870]]
+            ],
+                "notes": "Icarus became a State of Icenia on the 28th of August, 2023 via the Icarus-Icenia Merger",
+                "Icarus-Icenia Merger": "https://docs.google.com/document/d/1oAZ5I2xOZJKmqrnXxLQn_upj0CiG4K12VPffNcbVOYU/edit",
+                
+"nation": "Republic of Icenia",
+"website": "https://civicenia.github.io/",
+"color": "#00C9FF"
```

Comment:
Replace this with `{{ territory_details }}`

Reply:
ok, done.